modules/special-rules/pages/barrage.adoc

```diff
@@ -4,29 +4,54 @@ These weapons can saturate a wide area of the battlefield with intense salvoes o
 
 ---
 
-The more closely packed the units of the target detachment, the more effective a {barrage} weapon is likely to be against it.
+TIP: The more closely packed the units of the target detachment, the more effective {barrage} weapons are likely to be against it.
 This is simply because the barrage will have more target units within its area of effect.
 
-When you shoot with a unit that has a {barrage} weapon:
+When you make a {barrage} shot:
 
-. Take the Barrage template and place it over the target detachment.
-Place it such that it is over as many enemy units as possible.
+. Place a Barrage template in the standard way, but with these additional considerations:
+* Place only a single template, no matter how many {barrage} units in your detachment contribute to the attack.
+* Place the template so that it is completely within range and line of fire of *all* of the {barrage} units that contribute to the attack.
 +
-[NOTE]
-====
-You must cover at least three trooper models for an infantry unit to count or any part of a vehicle or war engine.
+You may opt to exclude some {barrage} units from the attack, but you may not make any separate attack with them against the same target detachment.
+* Cover as many units in the target detachment as possible.
+. Count the number of units of the target detachment that are under the template.
+This number is the Firepower value that each {barrage} unit contributes to the total Firepower that your detachment directs at the target detachment.
 
-+[TODO: It'd be good to rationalise this rule a bit, given that (1) infantry units don't always have five individual models on them, and (2) some people like to put their vehicles on bases, and (3) it isn't clear how much of a vehicle or war engine should count to resolve disagreement when a template barely 'nicks' such a unit.]+
+[IMPORTANT]
 ====
-. Count the number of units beneath the template.
-The number of units under the template is your unit's Firepower value for this shot.
-
-. Add this Firepower value to the rest of the Firepower of your detachment for the current attack. 
+When you place Blast markers, {barrage} weapons contribute to your detachment's total Firepower, but not to the number of super-heavy weapons.
 
-IMPORTANT: For the placement of Blast markers, count a {barrage} shot along with regular Firepower and not as a super-heavy weapon.
+It still takes only a single Blast marker to suppress a {barrage} weapon though, just like any other super-heavy weapon.
+====
 
```

Comment:
Something like this to clarify the intermingled case?

```suggestion

.Intermingled detachments under a Barrage template
[NOTE]
====
After you place to template to cover as many units in the target detachment as possible, it may be that the template also covers some units from another detachment.

Make a separate count of the number of units of the other detachment that are under the template.
After you have resolved the attack against the target detachment, use the separate count to resolve the attack against the other detachment, as if it were a separate attack.
====

```

One thing gives me pause for thought here. It almost feels like this should be in the general `barrage-templates.adoc` topic — except that we also need to refer to the "as many units in the target detachment as possible" bit here. Could use a partial of course, but I wonder if we can think of a neater way to resolve it? But we could look at that another time — I'd be happy just to clarify here for now, and maybe find a neat way to add the 'common' bit later.

Reply:
> Something like this to clarify the intermingled case?

Yes, that looks fine.

> One thing gives me pause for thought here. It almost feels like this should be in the general `barrage-templates.adoc` topic — except that we also need to refer to the "as many units in the target detachment as possible" bit here. Could use a partial of course, but I wonder if we can think of a neater way to resolve it? But we could look at that another time — I'd be happy just to clarify here for now, and maybe find a neat way to add the 'common' bit later.

But this doesn't, because it only applies to Barrage/Heavy Barrage! ;-)

All the other weapons that use the Barrage template hit the specific units under the template, rather than adding to a Firepower attack against specific detachments.

With the intermingled note, and a corrected example, I'm happy for this conversation to be marked as resolved.

Reply:
>But this doesn't, because it only applies to Barrage/Heavy Barrage! ;-)
>
>All the other weapons that use the Barrage template hit the specific units under the template

Ah, yes of course — doh!

Righto, I'll apply the above suggestion and resolve this conversation. We have another conversation on the example, so we still have a prompt to tackle that too.